Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -55,6 +56,12 @@ public int LaunchFile(string file, bool debug)
         {
             var nodePath = GetNodePath();
 
+            if(NpmInstallCommand.IsInstalling)
+            {
+                Nodejs.ShowNpmIsInstalling();
```

Comment:
>ShowNpmIsInstalling [](start = 23, length = 19)

Are we on an appropriate thread?  From below, I assume so, but it might be nice to document.